app/repositories/user.py

```
@@ -0,0 +1,163 @@
+            "password": get_password_hash(password),  # 해싱된 비밀번호
+            "nickname": nickname,
+            "profile_image": profile_image or "https://example.com/default-profile.jpg",
+            "created_at": now,
+            "updated_at": now
+        }
+        
+        return self.create(user_data)
+    
+    # ========== 수정 메서드 ==========
+    
+    def update_user(self, user_id: int, **updates) -> bool:
```

Ruff (0.14.11)

Missing type annotation for `**updates`

(ANN003)